modules/stanford_events_importer/tests/src/Kernel/Plugin/Field/FieldWidget/StanfordEventsImporterAPIURLFieldWidgetTest.php

```diff
@@ -18,17 +23,28 @@ class StanfordEventsImporterAPIURLFieldWidgetTest extends KernelTestBase {
   /**
    * {@inheritDoc}
    */
-  protected static $modules = [
+  public static $modules = [
     'system',
     'path_alias',
     'node',
     'user',
     'link',
     'datetime',
     'field',
-    'stanford_events_importer'
+    'field_permissions',
+    'migrate',
+    'migrate_tools',
+    'migrate_plus',
+    'field_group',
+    'config_pages',
+    'config_pages_overrides',
+    'stanford_events',
+    'stanford_events_importer',
   ];
 
+  // https://www.drupal.org/project/drupal/issues/3039565
+  protected $strictConfigSchema = FALSE;
```

Comment:
If I can get around the missing schema issues then perhaps.

Reply:
Couldn't get around this issue. Something about the test doesn't like that I am extending the link class and not providing a schema in the events module.

Reply:
lets make a ticket to review that schema later. its because you don't have a `modules/stanford_events_importer/config/schema/stanford_events_importer.schema.yml` file. alot like https://github.com/SU-SWS/stanford_media/blob/8.x-2.x/config/schema/stanford_media.schema.yml#L15

Reply:
Yeah, but I don't need/want a schema file. It is just the widget on an extended link field so I want the link field to continue to define the schema.

Reply:
its not a database schema. its a config schema.

Reply:
So even though I am only manipulating the form display widget in the UI and not adding any additional settings or storage Drupal still expects a copy of the settings here?